services/movie_session.py

```diff
@@ -0,0 +1,53 @@
+from db.models import MovieSession, CinemaHall, Movie
+from django.db.models import QuerySet
+from datetime import datetime
+
+
+def create_movie_session(movie_show_time: datetime, movie_id: int,
+                         cinema_hall_id: int) -> MovieSession:
+    movie = Movie.objects.get(id=movie_id)
+    cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)
+
+    new_session = MovieSession.objects.create(show_time=movie_show_time,
+                                              movie=movie,
+                                              cinema_hall=cinema_hall)
+
+    return new_session
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet:
```

Comment:
If argument can be None, you should show it in type hint
```suggestion
def get_movies_sessions(session_date: str | None = None) -> QuerySet:
```

Reply:
Fixed